### PR TITLE
Ensure every medium has it's own Counter.

### PR DESCRIPTION
**Pull request by esirK**
## Description
Ensure every medium has its own Counter to prevent identical values across all mediums.

Fixes # (issue)
Bug where all mediums have same values
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot 2021-04-07 at 11 13 46

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_builder.py [hunk 1]
```diff
@@ -3403,8 +3403,7 @@ def ws_107(self, ws):
                 for row in rows:
                     counts.update({(row[sq_field], TOPIC_GROUPS[row['topic']]): row['n']})
 
-                secondary_counts[media_type] = counts
-                
+                secondary_counts[media_type] = Counter(counts)
```
Comment (kilemensi) [inline reports/report_builder.py]: Why not leave
```suggestion
                secondary_counts[media_type] = counts
```
and just move the `counts` [initialization](https://github.com/CodeForAfrica/gmmp/pull/171/files#diff-19a686dbe88bb9f2f8c4f2ce5e331aa6f6db260926a2a5b7f1edfa7864a649b4L3392-L3393) inside the `for` loop:
```python
....
            for media_type, model in person_models.items():
                counts = Counter()
                sheet_name = model.sheet_name()
...
```
That way it's clear we need new `counts` for each media.
```

**Review by kilemensi — APPROVED** [review]
Good 👁️ @esirK .

-----------------------

Lets also check the other queries similar to this one (at the least the new ones i.e. **ws_104** to **ws_111**)

File: reports/report_builder.py (hunk 1 above)
Comment (esirK) [reply reports/report_builder.py]: yeah. This makes more sense.

**Comment by esirK** [comment]
@kilemensi  all the others have not been affected.

**Pull request merged**